### PR TITLE
Upload office (Office-31 + Caltech-10) and CMR dataset 

### PR DESCRIPTION
**Pull request by shuo-zhou**

### CODE REVIEW

File: cmr_images/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+|--|--|-- ... \
+|--|--|-- Phase_20.dcm \
+|--|-- Subject_2 \
+|--|-- Subject_3 \
+|--|-- … \
+|--|-- Subject_179 \
+|-- Mask \
+|--|-- Mask_WxH \
+|--|--|-- Phase_1.dcm \
+|-- Landmarks_WxH.csv
```
Comment (haipinglu) [inline cmr_images/ReadMe.md]: Also have class labels. And labels need some explanation.

File: cmr_images/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Cardiac MR Image (Short-Axis) Dataset for Pulmonary Arterial Hypertension Diagnosis
```
Comment (haipinglu) [inline cmr_images/ReadMe.md]: CMR should not be the top level structure. At least medical_images or else.

File: cmr_images/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+This dataset consists of the CMR images acquired from 179 subjects. The images have been processed in size (64x64) and (32x32). The images are shared in DICOM format.
+The files are organized in the following structure:
+
+|-- DICOM \
```
Comment (haipinglu) [inline cmr_images/ReadMe.md]: Is this the best way to show folder structure?

File: cmr_images/ReadMe.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Cardiac MR Image (Short-Axis) Dataset for Pulmonary Arterial Hypertension Diagnosis
+
+This dataset consists of the CMR images acquired from 179 subjects. The images have been processed in size (64x64) and (32x32). The images are shared in DICOM format.
```
Comment (haipinglu) [inline cmr_images/ReadMe.md]: Explain relationships with the reference at least briefly.

File: image_data/office/ReadMe.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Office dataset
+
+Office-31 + Caltech-10 dataset [[Original data source](http://www.stat.ucla.edu/~jxie/iFRAME/code/imageClassification.rar)]. Folder names (i.e. class names) of Caltech-10 from the original source have been renamed to be consistent with the folder names of Office-31.
+
+## Reference
+
+[1] Saenko, K., Kulis, B., Fritz, M. and Darrell, T., 2010, September. Adapting visual category models to new domains. In *European conference on computer vision* (pp. 213-226). Springer, Berlin, Heidelberg.
+
+[2] Griffin, Gregory and Holub, Alex and Perona, Pietro, 2007. Caltech-256 Object Category Dataset. California Institute of Technology. (Unpublished). https://resolver.caltech.edu/CaltechAUTHORS:CNS-TR-2007-001.
```
Comment (haipinglu) [inline image_data/office/ReadMe.md:13]: Are you sure this is the correct ref? http://www.vision.caltech.edu/Image_Datasets/Caltech101/

File: image_data/office/ReadMe.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Office dataset
+
+Office-31 + Caltech-10 dataset [[Original data source](http://www.stat.ucla.edu/~jxie/iFRAME/code/imageClassification.rar)]. Folder names (i.e. class names) of Caltech-10 from the original source have been renamed to be consistent with the folder names of Office-31.
```
Comment (haipinglu) [inline image_data/office/ReadMe.md]: Caltech-10 or Caltech-101?

File: cmr_images/ReadMe.md (hunk 2 above)
Comment (haipinglu) [reply cmr_images/ReadMe.md]: We should think of a better name for this dataset so others can refer to like those other popular dataset.

File: image_data/office/ReadMe.md (hunk 5 above)
Comment (haipinglu) [reply image_data/office/ReadMe.md:13]: See also https://www.cc.gatech.edu/~judy/domainadapt/

Comment (shuo-zhou) [reply image_data/office/ReadMe.md:13]: Please see the first paragraph in 4.1 and reference 15 of [Gong et al. (2012)](https://www.cs.utexas.edu/~grauman/papers/subspace-cvpr2012.pdf), which is also cited in Judy's link above.

File: image_data/office/ReadMe.md (hunk 6 above)
Comment (shuo-zhou) [reply image_data/office/ReadMe.md]: Caltech-10 is the 10 classes of Caltech-256 overlapped with Office-31. It is not the official name, but used in some papers.

File: cmr_images/ReadMe.md (hunk 2 above)
Comment (haipinglu) [reply cmr_images/ReadMe.md]: > CMR should not be the top level structure. At least medical_images or else.

under image

Comment (haipinglu) [reply cmr_images/ReadMe.md]: > We should think of a better name for this dataset so others can refer to lik

**ShefPAH179**

File: cmr_images/ReadMe.md (hunk 3 above)
Comment (haipinglu) [reply cmr_images/ReadMe.md]:
```sh
└───PAH_SA_32x32
    ├───DICOM
    │   ├───Subject_1
    │   ├───Subject_10
    │   ├───Subject_100
```

File: cmr_images/ReadMe.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+|-- Mask \
+|--|-- Mask_WxH \
+|--|--|-- Phase_1.dcm \
+|-- Landmarks_WxH.csv
+
+W, H denotes width and height of processed images, e.g. 64 or 32.
+
+## Reference
+
+Swift, A.J., Lu, H., Uthoff, J., Garg, P., Cogliano, M., Taylor, J., Metherall, P., Zhou, S., Johns, C.S., Alabed, S. and Condliffe, R.A., 2021. A machine learning cardiac magnetic resonance approach to extract disease features and automate pulmonary arterial hypertension diagnosis. *European Heart Journal-Cardiovascular Imaging*, 22(2), pp.236-245.
```
Comment (haipinglu) [inline cmr_images/ReadMe.md]: [1]

File: cmr_images/ReadMe.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+|--|-- … \
+|--|-- Subject_179 \
+|-- Mask \
+|--|-- Mask_WxH \
+|--|--|-- Phase_1.dcm \
+|-- Landmarks_WxH.csv
+
+W, H denotes width and height of processed images, e.g. 64 or 32.
+
+## Reference
```
Comment (haipinglu) [inline cmr_images/ReadMe.md]: Ask users to cite us explicitly like in http://www.vision.caltech.edu/Image_Datasets/Caltech101/

**Review by haipinglu — APPROVED** [review]

**Pull request merged**